pubsub/solace/amqp/metadata.yaml

```diff
@@ -0,0 +1,95 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: Solace-AMQP
+version: v1
+status: stable
+title: "solace-amqp"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-solace-amqp/
+capabilities:
+  - ttl
+authenticationProfiles:
+  - title: "Authenticate Using Solace properties"
+    description: "Solace authentication by providing url, username, and password."
+    metadata:
+      - name: url
+        required: true
+        sensitive: true
+        description: "Address of the AMQP broker. Can be secretKeyRef to use a secret reference."
+        example: '"amqp://host.domain[:port]"'
+        type: string
+      - name: username
+        required: true
+        sensitive: true
+        description: "The username to connect to the broker. Only required if anonymous is not specified or set to false."
+        example: '"default"'
+        type: string
+      - name: password
+        required: true
+        sensitive: true
+        description: "The password to connect to the broker. Only required if anonymous is not specified or set to false."
+        example: '"default"'  
+        type: string
+  - title: "Authenticate Using url and External TLS"
```

Comment:
TLS is not an auth profile. These properties can be used regardless of the auth profile